apps/src/applab/firebaseLogging.js

```diff
@@ -0,0 +1,45 @@
+import logToCloud from '../logToCloud';
+
+const LOG_DATA_TRANSFER_INTERVAL_MS = 1000;
+
+/**
+ * The timeout ID for the next flush of data transfer logs, if one is pending.
+ * @type {number}
+ */
+let flushDataTransferTimeout = null;
+
+/**
+ * The number of bytes transferred since the last flush.
+ * @type {number}
+ */
+let firebaseByteCount = 0;
+
+/**
+ * The number of write operations since the last flush.
+ * @type {number}
+ */
+let firebaseWriteCount = 0;
+
+/**
+ * Records that the specified number of bytes have been transferred to or from Firebase.
+ * This data is logged to NewRelic via addPageAction. A maximum of 20 page actions can
+ * be generated every 10 seconds, therefore we use batching and send the total number
+ * of bytes transferred at most once per second.
+ * @param {number} byteCount
```

Comment:
FWIW New Relic browser agent appears to batch whatever addPageAction calls you send to it, and only sends XHRs to the server once every 5-10 sec.

I'm glad you asked - in the past month or so it looks like Firebase has added a real-time usage dashboard! https://console.firebase.google.com/project/cdo-v3-prod/database/usage/last-30d/bandwidth
<img width="1072" alt="screen shot 2016-09-07 at 1 54 28 pm" src="https://cloud.githubusercontent.com/assets/8001765/18328190/a7ec42ba-7502-11e6-9879-2c36e1400e86.png">

This mostly obsoletes what I'm proposing in this PR, however what I have here would allow us to drill down a little and understand which apps are our problem apps. We still aren't any closer to a monitoring solution, though.

Thoughts on whether the monitoring in this PR is still worth adding given the discovery of the Firebase usage dashboard?